ci/run-sanity.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -ex
+
+rm -f config/run/init-completed
+
+timeout 15 ./run.sh &
+run=$!
+
+# shellcheck disable=SC2034
+for _i in $(seq 10)
+do
+  if [ -e config/run/init-completed ]
+  then
+    break
+  else
+    sleep 1
+  fi
+done
```

Comment:
How about this:
```bash
attempts=10
while [[ ! -f config/run/init-completed ]]; do
  sleep 1
   if ((--secs_to_next_genesis_poll == 0)); then 
    echo "Error: validator failed to boot"
    exit 1
  fi
done
```
If we don't add a timeout, on failure the script will never exit (unnecessarily consuming the CI machine until the overall build timeout occurs)